tests/ui/lang/control_flow/closure_capture_invalid_01.rs

```diff
@@ -0,0 +1,32 @@
+// build-pass
```

Comment:
```suggestion
// build-fail
```
(because you have a `.stderr` file and presumably compilation fails)